modules/jupyter-widget/package.json

```diff
@@ -29,13 +29,13 @@
     "prepublishOnly": "webpack -p"
   },
   "dependencies": {
-    "@deck.gl/aggregation-layers": "8.8.0-alpha.6",
-    "@deck.gl/carto": "8.8.0-alpha.6",
-    "@deck.gl/geo-layers": "8.8.0-alpha.6",
-    "@deck.gl/google-maps": "8.8.0-alpha.6",
-    "@deck.gl/json": "8.8.0-alpha.6",
-    "@deck.gl/layers": "8.8.0-alpha.6",
-    "@deck.gl/mesh-layers": "8.8.0-alpha.6",
+    "@deck.gl/aggregation-layers": "^8.8.0",
+    "@deck.gl/extensions": "^8.8.0",
```

Comment:
Why was this added?

Reply:
The package `@deck.gl/extensions` is a new peerDependency of `@deck.gl/carto` (https://github.com/visgl/deck.gl/blob/master/modules/carto/package.json#L59). I thought it would be interesting to have it for all the generic available in `@deck.gl/jupyter-widget`. 

However, if we don't want to add it for all, there is an alternative to be applied in `pydeck-carto` to load it dynamically for the custom layer using a combined load:

```
https://cdn.jsdelivr.net/combine/npm/@deck.gl/extensions@~8.8.*/dist.min.js,npm/@deck.gl/carto@~8.8.*/dist.min.js
```

Reply:
I finally removed this library and use the second method, which is more flexible. It can be added in the future if needed by a core module.